tuxemon/states/combat/combat.py

```diff
@@ -114,11 +114,13 @@ class TechniqueAnimationCache:
     def __init__(self) -> None:
         self._sprites: Dict[Technique, Optional[Sprite]] = {}
 
-    def get(self, technique: Technique) -> Optional[Sprite]:
+    def get(self, is_flipped: bool, technique: Technique) -> Optional[Sprite]:
```

Comment:
Fussy, but I would have these parameters the other way round. technique is the main parameter, is_flipped is secondary.

Reply:
That would make sense, especially since it's an extra flag. Changed!